src/augtool.c

```diff
@@ -481,7 +534,14 @@ static int main_loop(void) {
 
     while(1) {
         if (get_line) {
```

Comment:
you might wanna take a shot at refactoring it while you're extending it, because… well, you're adding more glue code onto it, and it's already hard to follow